src/codegen/codegen_ispc_visitor.cpp

```diff
@@ -249,8 +249,9 @@ void CodegenIspcVisitor::print_nrn_cur_matrix_shadow_reduction() {
     auto d_op = operator_for_d();
     if (info.point_process) {
         printer->add_line("uniform int node_id = node_index[id];");
-        printer->add_line(fmt::format("vec_rhs[node_id] {} shadow_rhs[id];", rhs_op));
-        printer->add_line(fmt::format("vec_d[node_id] {} shadow_d[id];", d_op));
+        printer->fmt_line("vec_rhs[node_id] {} shadow_rhs[id];", rhs_op);
+        printer->fmt_line("vec_rhs[node_id] {} shadow_rhs[id];", rhs_op);
```

Comment:
A mistake there, you copied the line twice

Reply:
Thanks, not sure how that slipped through (this was many regex search/replace, but I thought I'd checked...)